### PR TITLE
ROX-13758: Update FE Tests to fix UI CI failures

### PR DESCRIPTION
**Pull request by keyallis**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Review by clickboo — APPROVED** [review]
Would have loved the error message to be what it was changed from - since its more clear. Guess it can change after the UI test is fixed to expect it.

File: central/resourcecollection/service/service_impl.go [hunk 1]
```diff
@@ -166,7 +166,7 @@ func (s *serviceImpl) DeleteCollection(ctx context.Context, request *v1.Resource
 		return nil, errors.Wrap(err, "Failed to check for Report Configuration usages")
 	}
 	if reportConfigCount != 0 {
-		return nil, errors.Wrap(errox.ReferencedByAnotherObject, "Collection is in use by one or more report configurations")
+		return nil, errors.Wrap(errox.ReferencedByAnotherObject, "Failed to delete collection")
```
Comment (theencee) [inline central/resourcecollection/service/service_impl.go:169]: why not update the test? https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/vulnmanagement/reporting/reportingForm.test.js#L239

**Review by vjwilson — CHANGES_REQUESTED** [review]
Agree with Nakul--let's update the test instead.

**Review by theencee — APPROVED** [review]

**Comment by theencee** [comment]
@keyallis you should rebase. The style was fixed last night: https://github.com/stackrox/stackrox/pull/5413/files

**Review by vjwilson — APPROVED** [review]
FE change is OK

**Pull request merged**